### ISSUE TITLE
Name Tag - Badges

### ISSUE THREAD

**Issue by ChrisTheDBA** [ashumz/citycamp#247]
last year we had 4.5 x 7.5 holders.  having trouble finding one that will get here in time.

**Comment by ChrisTheDBA** [comment]
Check out these https://catalog.shopgoldner.com/product.cfm?uid=1901249&context=search&showInactive=N

**Comment by TaxLadyTracy** [comment]
Spoke with Shannon she is checking availablity and to verify it is actually what we are looking for.  
Once this is verified I will copy Jason to hopefully keep this all on 1 invoice.

**Comment by ChrisTheDBA** [comment]
badges have been ordered.  will be at Kapok Manor by Wednesday.

**Comment by jhibbets** [comment]
What is Kapok Manor?

**Comment by ChrisTheDBA** [comment]
Sorry that's my house. :)

**Comment by dtraleigh** [comment]
"like"

**Comment by ChrisTheDBA** [comment]
Yes our house sigil is a lactose intolerant direwolf rampant on a field of azure

**Comment by TaxLadyTracy** [comment]
Yeah not so sure about Chris sometimes!  BUT badges w/lanyards are here!!!
